src/components/Homepage/Homepage-categories.js

```diff
@@ -0,0 +1,29 @@
+import '../../css/home-page.css';
+import categoryRoom from '../../assets/categories/room-cat.jpg';
+import categoryDining from '../../assets/categories/dining-cat.jpg';
+import categoryHoliday from '../../assets/categories/holiday-cat.jpg';
```

Comment:
the assets like images or videos or any other file should be in the public folder of the project and then you can access it through the application by just saying <img src='/assets/...'  /> rather than importing everything in the file.

Reply:
Alright, will look into this.